glam/api/management/commands/import_desktop_aggs.py

```diff
@@ -124,3 +128,52 @@ def import_file(self, tmp_table, fp, model, channel):
                     percentiles = EXCLUDED.percentiles
             """
             cursor.execute(sql)
+
+    def import_revisions(self, channel):
+
+        FirefoxBuildRevisions = apps.get_model("api", "FirefoxBuildRevisions")
+
+        known_builds = list(
+            FirefoxBuildRevisions.objects.filter(channel=channel).values_list(
+                "build_id", flat=True
+            )
+        )
+        known_builds.append("*")
+
+        log(channel, f"We currently have {len(known_builds) - 1} known SHAs")
+
+        aggs_model = apps.get_model(CHANNEL_TO_MODEL[channel])
+        build_ids = list(
+            aggs_model.objects.exclude(build_id__in=known_builds)
+            .distinct("build_id")
+            .values_list("build_id", flat=True)
+        )
+
+        log(channel, f"We are missing {len(build_ids)} SHAs")
+
+        if len(build_ids) == 0:
+            log(channel, "No SHAs to update")
+            return
+
+        query = """
+            SELECT build.build.id, build.source.revision
+            FROM `moz-fx-data-shared-prod.telemetry.buildhub2`
+            WHERE build.build.id IN UNNEST(@build_ids)
+            AND build.target.channel = @channel
+            GROUP BY 1, 2
+        """
```

Comment:
if I want to see the output of this query in redash, what should I replace `@build_ids` and `@channel` with?